ariadne/asgi.py

```diff
@@ -167,6 +168,15 @@ async def render_playground(  # pylint: disable=unused-argument
     ) -> Response:
         return HTMLResponse(PLAYGROUND_HTML)
 
+    def build_json_response(
+        self,
+        content: Dict[str, Any],
+        status_code: int
+    ) -> Type[JSONResponse]:
```

Comment:
```suggestion
    ) -> Type[Response]:
```

Not that correct but makes possible inheritance chain more forgiving.